### DISCUSSION TITLE
Untouched beatmap with local label

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#24310] [Q&A] [answered]
Beatmap has `local` label to say it has been edited, but it's untouched

### Before deleting and re-downloading
![osu_2023-07-20_14-54-48](https://github.com/ppy/osu/assets/40531871/c4ee756f-ab71-4290-8442-0caedf3dd0e8)
### After downloading again
![osu_2023-07-20_14-55-46](https://github.com/ppy/osu/assets/40531871/1b1fe5a1-71a3-442d-9de1-98f24d4c07ed)

Why does it say locally edited 
[Beatmap](https://osu.ppy.sh/beatmapsets/881510#osu/1843086)

**Comment by peppy** [comment, marked as answer]
Tracking via https://github.com/ppy/osu/issues/22148
